package.json

```diff
@@ -23,11 +24,22 @@
     "url": "https://github.com/anyfin/bankid/issues"
   },
   "dependencies": {
-    "@types/node": "^12.11.5",
     "axios": "^0.21.1"
   },
   "devDependencies": {
-    "typescript": "^3.6.4",
-    "prettier": "^1.18.2"
+    "@types/node": "^14.14.22",
+    "typescript": "^4.1.3",
+    "prettier": "^2.2.1"
+  },
+  "peerDependencies": {
+    "@types/node": "^14.14.22"
+  },
+  "peerDependenciesMeta": {
```

Comment:
`@types/node` is only relevant for users on TypeScript. Marking it as an optional peer dependency skips its installation for users not on TS.